pkg/bgpv1/manager/reconciler/service.go

```diff
@@ -129,11 +129,6 @@ endpointsLoop:
 			continue
 		}
 
-		// We only need Endpoints tracking for externalTrafficPolicy=Local
```

Comment:
I am not sure if I understand why this is being removed ?

By removing it, all endpoints will be added in `ls (localServices)`. You might need to return localServices with internalTrafficPolicy=Local, ExternalTrafficPolicy=Local and error.

Reply:
My understanding is:

localServices should store all endpoints, and it should have nothing to do with what type of ExternalTrafficPolicy or InternalTrafficPolicy the svc has.

Reply:
Yes, you're right. I misunderstood. Logic to check local endpoints is at line 140.